pkg/docker/image_inspect.go

```diff
@@ -0,0 +1,29 @@
+package docker
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/docker/docker/api/types"
+)
+
+func ImageInspect(id string) (*types.ImageInspect, error) {
+	cmd := exec.Command("docker", "image", "inspect", id)
+	cmd.Env = os.Environ()
+
```

Comment:
```suggestion
console.Debug("$ " + strings.Join(cmd.Args, " "))
```

(github doesn't let me add the tab)

Reply:
Pushed with tab :)